### ISSUE TITLE
Anyone try out the new RX 5700(XT) cards?

### ISSUE THREAD

**Issue by iopq** [leela-zero/leela-zero#2475]
I am wondering if they can compete even without tensor cores.

**Comment by GoPlayers** [comment]
crash up to now

**Comment by roy7** [comment]
Here's someone with the same problem of LZ crashing on the 5700.

https://www.reddit.com/r/cbaduk/comments/ctk67p/leelaz_gpu_hardware_problem/

**Comment by GoPlayers** [comment]
I leave a log from sabaki, v0.17
[rx5700_prblm.txt](https://github.com/leela-zero/leela-zero/files/3526770/rx5700_prblm.txt)

**Comment by ihavnoid** [comment]
Since I don't have the hardware my capability of debugging anything is very limited, but this just looks like the OpenCL is crashing.

@GoPlayers  Is this Windows or Linux?  Can you try '--precision single' ?

**Comment by GoPlayers** [comment]
it's windows 10. I set single precision, I made 20-30 moves and didn't crushed. But, this config a little slow I think. 
Do you think in new ver. of leela or Radeon driver will solve that problem?

In RX360 GPU in the opencl_tuning file there was  XgemmBatchedHalf and XgemmBatched. But in RX5700 there was only one setting. rx360 was tune 2 times when starting. may be important :) 
thank you,

After change 2 threats to 4, tuned again and crushed :(

**Comment by GoPlayers** [comment]
I change my RX5700 to RTX2060. Perhaps driver problems I don't know, RTX2060 faster than rx. I will give some first move stats.
 RX5700:
10.3 average depth, 42 max depth
15973 non leaf nodes, 1.21 average children
19296 visits, 6784631 nodes, 19295 playouts, 2134 n/s

RTX2060:
11.5 average depth, 39 max depth
32905 non leaf nodes, 1.24 average children
40872 visits, 14323109 nodes, 40871 playouts, 4353 n/s
Fast as double isn't it ?

**Comment by iopq** [comment]
First move stats are not accurate. They use symmetry, so 4x on 4-4 and 8x on 3-4

--bechmark plays a few moves to break symmetry, run it with that instead

**Comment by GoPlayers** [comment]
Yes you write. Quantities too high (playouts 10 times) but I think ratio will be the same. 
Now, I couldn't measure again. I gave back the rx card.

**Comment by MSD2000** [comment]
GoPlayers, what network did you use in this test? 15 blocks? 4353 visits per second

**Comment by GoPlayers** [comment]
40 I think, LZ242 (for first move 4353 is not accurate)

Now with RTX 2060, for 5th move of the game:
34.5 average depth, 71 max depth
5613 non leaf nodes, 1.21 average children
6772 visits, 2186265 nodes, 5005 playouts, 354 n/s
R17 ->    5552 (V: 58.11%) (LCB: 57.98%) (N: 47.91%) PV: R17 Q17 R16 R14 R15 Q15 S14 R18 S18 S15 Q18 S17 R19 S16 R18 S13 Q14 R13 O17 O14 M16 C17 D17 C16 D15 B14 P14 P13 P15 Q13 K3 N16 S5 R4 R7 R6 S6 S8 S7 P8 R9 S9 P9 O9

**Comment by MSD2000** [comment]
GoPlayers, thanks for the info!

**Comment by iopq** [comment]
@GoPlayers you're getting half of the speed you should be getting. Is half precision on? 10 threads 5 batches?

**Comment by GoPlayers** [comment]
I run again after deleting tuning file. Again 10 threads 5 batches, tune file ; XgemmBatchedHalf set, but, I didn't set half precision. Speed look like double now, like 733 n/s. I give log:

[sabaki_2019-08-30-08-16-43_12916.log](https://github.com/leela-zero/leela-zero/files/3558374/sabaki_2019-08-30-08-16-43_12916.log)

Inform me also :)

**Comment by iopq** [comment]
Anyone try it on Linux? There's some suspicion that this is related to Windows drivers
